src/components/POLine/POLineView.js

```diff
@@ -265,7 +265,7 @@ class POLineView extends Component {
         >
           <Row end="xs">
             <Col xs={10}>
-              {isOrderClosed && (
+              {isClosedOrder && (
```

Comment:
so cool change, so meaningful, wow
![image](https://user-images.githubusercontent.com/43577432/81171075-72b38e80-8fa4-11ea-8709-123d4f712b74.png)

Reply:
just to be consistent in the app 😂

Reply:
but it was here before, so according to your logic you should been using it in other places for consistency. For now it's just a meaningless diff